app/assets/javascripts/MathJax/mathjax_helpers.js

```diff
@@ -13,23 +13,31 @@ function hideAnnotationPreview() {
     dialog.css('margin-left', -1 * dialog.width() / 2);
 }
 
-function showAnnotationPreview() {
-    $('#annotation_preview').show();
-    $('#annotation_preview_title').show();
-
-    // recenter dialog
-    var dialog = $('#annotation_dialog');
-    dialog.css('margin-left', -1 * dialog.width() / 2);
-}
-
 function updateAnnotationPreview() {
     var newAnnotation = document.getElementById('new_annotation_content').value;
 
     var preview = document.getElementById('annotation_preview');
     preview.innerHTML = marked(newAnnotation);
 
-    showAnnotationPreview();
-
     // typeset the preview
     MathJax.Hub.Queue(['Typeset', MathJax.Hub, preview]);
 }
+
+// Allow inline single dollar sign notation
+MathJax.Hub.Config({
+  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
+});
+
+var delay = (function(){
+  var timer = 0;
+  return function(callback, ms){
+    clearTimeout (timer);
+    timer = setTimeout(callback, ms);
+  };
+})();
+
+$(document).on("keyup", "#new_annotation_content", function () {
+  delay(function(){
```

Comment:
So this, at least, should be possible to simplify just by passing `updateAnnotationPreview`.